### ISSUE TITLE
openvpn - unable to contact daemon, service not running?

### ISSUE THREAD

**Issue by ElXk6** [opnsense/core#5168]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

[X] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md

[X] I have searched the existing issues and I'm convinced that mine is new.

[X] The title contains the plugin to which this issue belongs

**Describe the bug**
One of two VPN-Server closed from yesterday to today with following failure.
openvpn.log:
   Options error: --client-connect requires --mode server

Error in the GUI:
   unable to contact daemon, service not running?

There was only a running service for the working VPN. Also, no Sockets.
VPN restart did nothing.

I diffed both server configs and found out, that the not working one had no "server <IP> <subnet mask>" entry.
But in the GUI it was set. So I saved it again, delete the config and saved it again, but both didn't work.
The only solution was to add it myself. After this, the VPN was working again.

**Relevant log files**
```
/conf/config.xml diff from a week ago:
<     <username>(system)</username>
<     <time>1627620670.4227</time>
<     <description>/usr/local/opnsense/mvc/script/run_migrations.php made changes</description>
---
>     <username>XXX</username>
>     <time>1628158692.6676</time>
>     <description>/vpn_openvpn_server.php made changes</description>
1841,1842c1841,1842
<       <vxlans version="1.0.1"/>
<       <loopbacks version="1.0.0"/>
---
>       <vxlans/>
>       <loopbacks/>
2366d2365
<       <engine>none</engine>
2370,2371d2368
<       <dns_domain>xx.xx.xx</dns_domain>
<       <dns_server1>192.168.xxx.xxx</dns_server1>
2373c2370
<       <verbosity_level>1</verbosity_level>
---
>       <verbosity_level>3</verbosity_level>
2448c2445
<   <staticroutes version="1.0.0"/>
---
>   <staticroutes/>
```

**Environment**
Software version used and hardware type if relevant.
e.g.:

OPNsense 21.7.1-amd64
FreeBSD 12.1-RELEASE-p19-HBSD
OpenSSL 1.1.1k 25 Mar 2021

---

**Question**
Is this a known bug?
Or was it something on my side?

**Comment by Napsterbater** [comment]
Is this going to be included in 21.7.2? I see it is currently marked for 22.1

**Comment by fichtner** [comment]
Because 21.7 milestone is closed because it’s been released.
